src/js/actions/search/layers.js

```diff
@@ -131,7 +129,8 @@ define(function (require, exports) {
      *
      */
     var registerLayerSearch = function () {
-        var filters = Immutable.List(["PIXEL", "TEXT", "ARTBOARD", "ADJUSTMENT", "SMARTOBJECT", "GROUP", "VECTOR"]);
+        var filters = Immutable.List(
+                ["LAYER", "PIXEL", "TEXT", "ARTBOARD", "ADJUSTMENT", "SMARTOBJECT", "GROUP", "VECTOR"]);
```

Comment:
Would look better if the closing parentheses was in a new line with same indent as the var